docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -33,27 +41,32 @@ Between May 13 2024 and August 6 2024, [signal values were much lower](#limitati
 * _s04_: Shortness of breath, Wheeze, Croup, Pneumonia, Asthma, Crackles, Acute bronchitis, Bronchitis
 * _s05_: Anosmia, Dysgeusia, Ageusia
 * _s06_: Laryngitis, Sore throat, Throat irritation
+* _s07_: Conjunctivitis, Red eye, Epiphora, Eye pain, Rheum
 * _scontrol_: Type 2 diabetes, Urinary tract infection, Hair loss, Candidiasis, Weight gain
 
-The symptoms were combined in sets that showed positive correlation with cases, especially after Omicron was declared a variant of concern by the WHO. Note that symptoms in _scontrol_ are not COVID-19 related, and this symptom set can be used as a negative control.
+The symptoms were combined in sets _s01_-_s06_ that showed positive correlation with COVID-19 cases, especially after Omicron was declared a variant of concern by the WHO.
```

Comment:
the sentence structure feels weird here...  im not sure if my edit is still accurate though?
```suggestion
The symptoms combined in sets _s01_-_s06_ show positive correlation with COVID-19 cases, especially after Omicron was declared a variant of concern by the WHO.
```